### ISSUE TITLE
[Screen reader - Logic Apps - Workflow settings]: "Select an Integration account" & "Run history retention in days" Labels are not associated with their combo boxes.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6111]
### Describe the issue 

User Experience:

 Users who rely on screen reader for navigation will be impacted if screen reader does not announce label information along with its combo box as users will not get complete information of the control and they will be misguided.

Note: User credentials should NOT be included in the bug.

Repro Steps:

- Open [https://ms.portal.azure.com/](<https://ms.portal.azure.com/>) and login via valid credentials. 
- Navigate to the Left Nav bar and select 'More Services', search 'Logic Apps' and select it. Select any already created logic app. 
- Select 'Work Flow Settings' in the left navigation pane. 
- Navigate all the controls on the 'Work Flow Settings' blade. 
- Verify whether "Select an Integration account" & "Run history retention in days" Labels are associated with their combo boxes. 

Actual Result:

 "Select an Integration account" & "Run history retention in days" Labels are not associated with their combo boxes.

 Observation:

 Screen reader announces information as "dash combo box collapsed opens list".

Expected Result:

 "Select an Integration account" & "Run history retention in days" Labels should be associated with their combo boxes.

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

*No response*

### Browser 

Chrome, Edge

### Additional context 

*No response*
